scripts/waffles.coffee

```diff
@@ -37,6 +38,17 @@ module.exports = (robot) ->
           order_name != name and not order_name.endsWith " _via #{name}_"
       robot.brain.set waffleType, nameList
 
+  endOrder = (msg) ->
+    totalWaffles = (robot.brain.get(waffleType).length for waffleType in waffleTypes)
+      .reduce (x,y)->x+y
```

Comment:
Minor nit: ruby style usually wants [spaces around operators](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

Reply:
Woot this is very atypical of me haha. Ok I corrected it.